config/config.json

```diff
@@ -9,45 +9,174 @@
+        "security.FloatLoopCounter" : ["default", "sensitive", "extreme"],
+        "security.insecureAPI.UncheckedReturn" : ["default", "sensitive", "extreme"],
+        "security.insecureAPI.getpw" : ["default", "sensitive", "extreme"],
+        "security.insecureAPI.gets" : ["default", "sensitive", "extreme"],
+        "security.insecureAPI.mkstemp" : ["default", "sensitive", "extreme"],
+        "security.insecureAPI.mktemp" : ["default", "sensitive", "extreme"],
+        "security.insecureAPI.rand" : ["default", "sensitive", "extreme"],
+        "security.insecureAPI.strcpy" : ["sensitive", "extreme"],
+        "security.insecureAPI.vfork" : ["default", "sensitive", "extreme"],
+        "unix.API" : ["default", "sensitive", "extreme"],
+        "unix.Malloc" : ["default", "sensitive", "extreme"],
+        "unix.MallocSizeof" : ["default", "sensitive", "extreme"],
+        "unix.MismatchedDeallocator" : ["default", "sensitive", "extreme"],
+        "unix.StdCLibraryFunctions" : ["default", "sensitive", "extreme"],
+        "unix.Vfork" : ["default", "sensitive", "extreme"],
+        "unix.cstring.BadSizeArg" : ["default", "sensitive", "extreme"],
+        "unix.cstring.NullArg" : ["default", "sensitive", "extreme"],
+        "valist.CopyToSelf" : ["default", "sensitive", "extreme"],
+        "valist.Uninitialized" : ["default", "sensitive", "extreme"],
+        "valist.Unterminated" : ["default", "sensitive", "extreme"]
+      },
+      "clang-tidy_checkers" : {
+        "bugprone-suspicious-memset-usage" : ["default", "sensitive", "extreme"],
+        "bugprone-undefined-memory-manipulation" : ["default", "sensitive", "extreme"],
+        "cert-dcl03-c" : ["default", "sensitive", "extreme"],
+        "cert-dcl21-cpp" : ["default", "sensitive", "extreme"],
+        "cert-dcl50-cpp" : ["default", "sensitive", "extreme"],
+        "cert-dcl54-cpp" : ["default", "sensitive", "extreme"],
+        "cert-dcl58-cpp" : ["default", "sensitive", "extreme"],
+        "cert-dcl59-cpp" : ["default", "sensitive", "extreme"],
+        "cert-env33-c" : ["default", "sensitive", "extreme"],
+        "cert-err09-cpp" : ["extreme"],
+        "cert-err34-c" : ["extreme"],
+        "cert-err52-cpp" : ["default", "sensitive", "extreme"],
+        "cert-err58-cpp" : ["sensitive", "extreme"],
+        "cert-err61-cpp" : ["default", "sensitive", "extreme"],
+        "cert-fio38-c" : ["default", "sensitive", "extreme"],
+        "cert-flp30-c" : ["default", "sensitive", "extreme"],
+        "cert-msc30-c" : ["default", "sensitive", "extreme"],
+        "cert-msc50-cpp" : ["default", "sensitive", "extreme"],
+        "cert-oop11-cpp" : ["default", "sensitive", "extreme"],
+        "cppcoreguidelines-c-copy-assignment-signature" : ["default", "sensitive", "extreme"],
+        "cppcoreguidelines-interfaces-global-init" : ["sensitive", "extreme"],
+        "cppcoreguidelines-no-malloc" : ["default", "sensitive", "extreme"],
```

Comment:
I would not turn on this by default. The users might use CodeChecker with C codebase which would have malloc as well. And this might also be very noisy on legacy C codebases.